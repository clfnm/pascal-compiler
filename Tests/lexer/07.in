i := 10(* hello, world *) i : = 10 { 123 } I:=10
// hello
x += 10;