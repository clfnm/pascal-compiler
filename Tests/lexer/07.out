1	1	Id	i	i
1	3	Operation	Assigment	:=
1	6	Integer	10	10
1	27	Id	i	i
1	29	Separator	Colon	:
1	31	Operation	Equal	=
1	33	Integer	10	10
1	44	Id	I	I
1	45	Operation	Assigment	:=
1	47	Integer	10	10
3	1	Id	x	x
3	3	Operation	AdditionWithAssigment	+=
3	6	Integer	10	10
3	8	Eof	EOF	